code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -0,0 +1,395 @@
+/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash
+	name = "Ashen passage"
+	desc = "Low range spell allowing you to pass through a few walls."
```

Comment:
make sure that this spell won't make that distinctive jaunting noise when its effect ends, or else that'll kind of blow your stealth

Reply:
good catch